docs/api/IoT.js-API-N-API.md

```diff
@@ -0,0 +1,238 @@
+| [napi_make_callback](https://nodejs.org/api/n-api.html#n_api_napi_make_callback)  | O |
+| [napi_new_instance](https://nodejs.org/api/n-api.html#n_api_napi_new_instance)  | O |
+| [napi_open_callback_scope](https://nodejs.org/api/n-api.html#n_api_napi_open_callback_scope)  | O |
+| [napi_open_escapable_handle_scope](https://nodejs.org/api/n-api.html#n_api_napi_open_escapable_handle_scope)  | O |
+| [napi_open_handle_scope](https://nodejs.org/api/n-api.html#n_api_napi_open_handle_scope)  | O |
+| [napi_property_attributes](https://nodejs.org/api/n-api.html#n_api_napi_property_attributes)  | O |
+| [napi_property_descriptor](https://nodejs.org/api/n-api.html#n_api_napi_property_descriptor)  | O |
+| [napi_queue_async_work](https://nodejs.org/api/n-api.html#n_api_napi_queue_async_work)  | O |
+| [napi_ref](https://nodejs.org/api/n-api.html#n_api_napi_ref)  | O |
+| [napi_reference_ref](https://nodejs.org/api/n-api.html#n_api_napi_reference_ref)  | O |
+| [napi_reference_unref](https://nodejs.org/api/n-api.html#n_api_napi_reference_unref)  | O |
+| [napi_reject_deferred](https://nodejs.org/api/n-api.html#n_api_napi_reject_deferred)  | O |
+| [napi_remove_env_cleanup_hook](https://nodejs.org/api/n-api.html#n_api_napi_remove_env_cleanup_hook)  | O |
+| [napi_remove_wrap](https://nodejs.org/api/n-api.html#n_api_napi_remove_wrap)  | O |
+| [napi_resolve_deferred](https://nodejs.org/api/n-api.html#n_api_napi_resolve_deferred)  | O |
+| [napi_set_element](https://nodejs.org/api/n-api.html#n_api_napi_set_element)  | O |
+| [napi_set_named_property](https://nodejs.org/api/n-api.html#n_api_napi_set_named_property)  | O |
+| [napi_set_property](https://nodejs.org/api/n-api.html#n_api_napi_set_property)  | O |
+| [napi_status](https://nodejs.org/api/n-api.html#n_api_napi_status)  | O |
+| [napi_strict_equals](https://nodejs.org/api/n-api.html#n_api_napi_strict_equals)  | O |
+| [napi_throw](https://nodejs.org/api/n-api.html#n_api_napi_throw)  | O |
+| [napi_throw_error](https://nodejs.org/api/n-api.html#n_api_napi_throw_error)  | O |
+| [napi_throw_range_error](https://nodejs.org/api/n-api.html#n_api_napi_throw_range_error)  | O |
+| [napi_throw_type_error](https://nodejs.org/api/n-api.html#n_api_napi_throw_type_error)  | O |
+| [napi_typedarray_type](https://nodejs.org/api/n-api.html#n_api_napi_typedarray_type)  | O |
+| [napi_typeof](https://nodejs.org/api/n-api.html#n_api_napi_typeof)  | O |
+| [napi_unwrap](https://nodejs.org/api/n-api.html#n_api_napi_unwrap)  | O |
+| [napi_value](https://nodejs.org/api/n-api.html#n_api_napi_value)  | O |
+| [napi_valuetype](https://nodejs.org/api/n-api.html#n_api_napi_valuetype)  | O |
+| [napi_wrap](https://nodejs.org/api/n-api.html#n_api_napi_wrap)  | O |
+
+
+# N-API
+
+N-API (pronounced N as in the letter, followed by API) is an API for building native Addons. N-API is generally used to create and manipulate JavaScript values. You can read more about N-API [here](https://nodejs.org/api/n-api.html#n_api_n_api).
```

Comment:
The link should point to the LTS version not to the latest.

Reply:
Right, changed all of the links to the LTS version.